pom.xml

```diff
@@ -37,6 +37,26 @@
             <artifactId>jackson-mapper-asl</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+
+        <!-- unit test dependencies -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit-dep</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hamcrest</groupId>
+                    <artifactId>hamcrest-core</artifactId>
```

Comment:
For some reason the JUnit jar includes a copy of Hamcrest within it, but it's an old version, so this exclusion prevents it from overriding the current version of Hamcrest that there's a separate dependency for.